packages/react-proptypes/src/editorOptions.spec.js

```diff
@@ -0,0 +1,9 @@
+import editorOptions from './editorOptions';
+
+test(`should export a function.`, () => {
+    expect(typeof (editorOptions)).toBe('function');
```

Comment:
no braces necessary, just `typeof editorOptions`

Reply:
I'm pretty sure I just copied that from somewhere… ;)